app/controllers/orders_controller.rb

```diff
@@ -0,0 +1,68 @@
+class OrdersController < ApplicationController
+   rescue_from ActiveRecord::RecordNotFound, with: :catch_not_found
+     before_action :set_order, only: [ :show, :edit, :update, :destroy ]
+
+      # GET /orders or /orders.json
+     def index
+      @orders = Order.all
+     end
+
+   # GET /orders/1 or /orders/1.json
+   def show
+   end
+
+  # GET /orders/new
+   def new
+    @order = Order.new
+  end
+
+   # GET /orders/1/edit
+  def edit
+  end
+
+# POST /orders or /orders.json
+  def create
+    @order = Order.new(order_params)
+    if @order.save
+      flash.notice = "The order record was created successfully."
+      redirect_to @order
+    else
+      render :new, status: :unprocessable_entity
+    end
+  end
+
+      # PATCH/PUT /orders/1 or /orders/1.json
+  def update
+    debugger
+    if @order.update(order_params)
+      flash.notice = "The order record was updated successfully."
+      # redirect_to @order
+    else
+      render :edit, status: :unprocessable_entity
+    end
+  end
+
+  # DELETE /orders/1 or /orders/1.json
+  def destroy
+    @order.destroy
+    respond_to do |format|
+      format.html { redirect_to orders_url, notice: "Order was successfully destroyed." }
+      format.json { head :no_content }
+    end
+  end
+
+   private
+    # Use callbacks to share common setup or constraints between actions.
+    def set_order
+      @order = Order.find(params[:id])
+    end
+
+    def order_params
+      params.require(:order).permit(:product_name, :product_count, :customer_id)
+    end
+     def catch_not_found(e)
+      Rails.logger.debug("We had a not found exception.")
+      flash.alert = e.to_s
+      redirect_to orders_path
+end
```

Comment:
nitpick note: watch your indentation, you want this to be consistent and have opening `def`s line up with closing `end`s.

I'd recommend looking into a tool called [Rubocop](https://rubocop.org/) - it's a tool you can use and run easily from the command line to format your files and make syntax suggestions.